File: Sorting/Insertion Sort/Javascript/InsertionSort.js
function insertionSort (items) {
    for (var i = 0; i < items.length; i++) {
      let value = items[i]
      for (var j = i - 1; j > -1 && items[j] > value; j--) {
        items[j + 1] = items[j]
      }
      items[j + 1] = value
    }
  
    return list
  }
  
  const list = [54, 26, 93, 17, 77, 31, 44, 55, 20]
  console.log(insertionSort(list))

File: Sorting/Insertion Sort/Python/insertion_sort.py
def insertion_sort(arr):
	'''
		Args: Input list
		Returns: Sorted list
	'''
	for i in range(1, len(arr)):
		key = arr[i]
		j = i-1
		# finding element greater than current value
		while(j>=0 and arr[j]>key):
			arr[j+1] = arr[j]
			j -= 1
		arr[j+1] = key

	return arr

# Driver Program
if __name__ == '__main__':
	arr = [5,3,9,6,1,8]
	arr = insertion_sort(arr)
	print(arr)